README.md

```diff
@@ -2,90 +2,276 @@
+- Sub-window, multi-window, and out-of-process rendering.
+- React Native integration.
+
+This section will be updated frequently. If you have any questions, please reach out
+to us on [the Babylon forum](https://forum.babylonjs.com).
 
 ## Getting Started
 
-### Windows
-
-#### Prerequisites
-
-- [CMake](https://cmake.org/download/) 3.12 or higher
-- [Python](https://www.python.org/downloads/) 3.x
-- [Visual Studio](https://visualstudio.microsoft.com/downloads/) 2019 (16.3 or higher)
-
-#### Build
-
-- Clone this repo.
-- Update the submodules.
-  ```
-  C:\BabylonNative>git submodule update --init --recursive
-  ```
-- Create a new directory for the build files, e.g. `Build` at the root of the repo.
-  ```
-  C:\BabylonNative>mkdir Build
-  ```
-- Change your working directory to the new directory.
-  ```
-  C:\BabylonNative>cd Build
-  ```
-- Run CMake from the new directory and point to the root of the repo.
-
-    NOTE: If you get the CMake error *No preprocessor test for "Intel"*, update your Visual Studio to 16.3 or higher.
-  ```
-  C:\BabylonNative\Build>cmake ..
-  ```
-- Open the generated solution `BabylonNative.sln`.
-  ```
-  C:\BabylonNative\Build>start BabylonNative.sln
-  ```
-- Once the solution opens, right click the Playground project, click "Set as StartUp Project" and click the play button at the top to run the test app which should render a box
-
-### Android / iOS / MacOS
-
-*Planned but not yet implemented*
+Welcome to Babylon! This quick overview will help you get started developing in the 
+Babylon Native repository. We support development on Windows and macOS. (It is also
+possible, though not yet fully supported, to develop on Linux.) This overview is 
+intended for reasonably experienced developers with familiar with common native 
+development principles.
+
+### **All Development Platforms, Common First Steps**
+
+**Required Tools:** [git](https://git-scm.com/), [CMake](https://cmake.org/)
+
+Step 1 for all development environments and targets is to clone the repo. Use a 
+git-enabled terminal to follow the steps below.
+
+```
+git clone https://github.com/BabylonJS/BabylonNative.git
+```
+
+Babylon Native makes extensive use of submodules to supply its dependencies, so it's
+also necessary to set up the submodules.
+
+```
+cd BabylonNative
+git submodule update --init --recursive
+```
+
+Babylon Native's build system is based on CMake, which customarily uses a separate
+build directory. Build directory location is up to you, but we recommend just creating 
+a `Build` directory within your clone of the Babylon Native repository (Babylon 
+Native's `.gitignore` file is already set up to ignore this `Build` directory).
+
+```
+mkdir Build
+cd Build
+```
+
+### **Building on Windows 10, Targeting Windows Desktop (Win32)**
+
+**Required Tools:** [Visual Studio 2019](https://visualstudio.microsoft.com/vs/) with 
+C++ development tools, [Python 3.0](https://www.python.org/) or newer (required by 
+dependencies)
```

Comment:
Have we confirmed Python is actually necessary?

Reply:
There are numerous Python scripts in the bgfx submodule, but I haven't actually confirmed what happens if you try to build without Python. I could check that now, or we could do it later; I've created an issue (#161) to track it either way. Thoughts?

Reply:
Not that big a deal either way.